code/mob/living/intangible/ai-camera.dm

```diff
@@ -58,34 +58,25 @@
 		src.client.show_popup_menus = 1
 		var/client_color = src.client.color
 		src.client.color = "#000000"
-		SPAWN(0) //let's try not hanging the entire server for 6 seconds every time an AI has wonky internet
-			if (!src.client) // just client things
-				return
-			src.client.images += aiImages
-			src.bioHolder.mobAppearance.pronouns = src.client.preferences.AH.pronouns
-			src.update_name_tag()
-			src.job = "AI"
-			if (src.mind)
-				src.mind.assigned_role = "AI"
-			animate(src.client, 0.3 SECONDS, color = client_color)
-			var/sleep_counter = 0
-			for(var/image/I as anything in aiImagesLowPriority)
-				src.client << I
-				if(sleep_counter++ % (300 * 10) == 0)
-					LAGCHECK(LAG_LOW)
+		if (!src.client) // just client things
+			return
+		src.bioHolder.mobAppearance.pronouns = src.client.preferences.AH.pronouns
+		src.update_name_tag()
+		src.job = "AI"
+		if (src.mind)
+			src.mind.assigned_role = "AI"
+		animate(src.client, 0.3 SECONDS, color = client_color)
+		SPAWN(0)
```

Comment:
Could maybe remove the `animate` and `SPAWN` since they're meant to cover up the time during the lagspike when the images haven't loaded yet.